hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/metrics/Metric.java

```diff
@@ -71,9 +72,4 @@
      */
     void set(long newValue);
 
-    /**
-     * Returns the current value.
-     */
-    long get();
```

Comment:
Actually this means that value of User Metric cannot be obtained in user code. Doesn't it too restrictive? I believe there can be scenarios where it is useful to `get` current value, modify it and `set` back to metric.

Reply:
Doing that would be not safe, just a simple `get` method doesn't allow you to do it atomically. Better to not have it at all.